File: hcgcalidadapp/lib/src/modelos/actividad.dart
import 'dart:convert';

Actividad actividadFromJson(String str) => Actividad.fromJson(json.decode(str));

String actividadToJson(Actividad data) => json.encode(data.toJson());

class Actividad {
  Actividad(
      {this.actividadId,
      this.actividadUsuarioControlId,
      this.actividadDetalle,
      this.actividadHoraInicio,
      this.actividadHoraFin,
      this.actividadFecha,
      this.postcosechaId,
      this.tipoActividadId,
      this.tipoActividadDescripcion
      //this.
      });

  int actividadId;
  int actividadUsuarioControlId;
  String actividadDetalle;
  String actividadHoraInicio;
  String actividadHoraFin;
  DateTime actividadFecha;
  int postcosechaId;
  int tipoActividadId;
  String tipoActividadDescripcion;

  factory Actividad.fromJson(Map<String, dynamic> json) => Actividad(
        actividadId: json["actividadId"],
        actividadUsuarioControlId: json["actividadUsuarioControlId"],
        actividadDetalle: json["actividadDetalle"],
        actividadHoraInicio: json["actividadHoraInicio"],
        actividadHoraFin: json["actividadHoraFin"],
        postcosechaId: json["postcosechaId"],
        tipoActividadId: json["tipoActividadId"],
        tipoActividadDescripcion: json["tipoActividadDescripcion"],
        actividadFecha: DateTime.parse(json["actividadFecha"]),
      );

  Map<String, dynamic> toJson() => {
        "actividadId": actividadId,
        "actividadUsuarioControlId": actividadUsuarioControlId,
        "actividadDetalle": actividadDetalle,
        "actividadHoraInicio": actividadHoraInicio,
        "actividadHoraFin": actividadHoraFin,
        "postcosechaId": postcosechaId,
        "tipoActividadId": tipoActividadId,
        "tipoActividadDescripcion":tipoActividadDescripcion,
        "actividadFecha": actividadFecha.toIso8601String(),
      };
}

File: hcgcalidad_services-master/hcgcalidad_services-master/HCGCALIDADSERVICES/Models/Actividad.cs
using System;
using System.Collections.Generic;

namespace HCGCALIDADSERVICES.Models
{
    public partial class Actividad
    {
        public int ActividadId { get; set; }
        public string ActividadDetalle { get; set; }
        public string ActividadHoraInicio { get; set; }
        public string ActividadHoraFin { get; set; }
        public DateTime? ActividadFecha { get; set; }
        public int? UsuarioControlId { get; set; }
        public int? PostcosechaId { get; set; }
        public int? TipoActividadId { get; set; }
        public Postcosecha Postcosecha { get; set; }
        public Usuariocontrol UsuarioControl { get; set; }
        public TipoActividad TipoActividad { get; set; }

    }
}
